### DISCUSSION TITLE
add all msys2 environments to repology.org

### DISCUSSION THREAD

**Discussion by ognevny** [msys2/MINGW-packages#20509] [Ideas]
currently at repology there are 2 environments: MINGW32 (mingw-i686) and MINGW64 (mingw-x86_64). MINGW32 packages are <50% of all, some new packages don't have both

**Comment by Biswa96** [comment]
Good idea. We could suggest them to change that ucrt64 and clang64 (or any one of them). Adding all environments would be unnecessary.

**Reply by ognevny** [reply]
and clangarm64

**Reply by Biswa96** [reply]
If I am not missing something all the packages in clangarm64 already exist in clang64.

**Comment by ognevny** [comment]
https://github.com/repology/repology-updater/issues/1400

**Comment by ognevny** [comment]
https://github.com/repology/repology-updater/commit/1cb1deb4b7ea3d9de4ee162d015def4ed6198dd9
